cmd/eskip/args_test.go

```diff
@@ -81,7 +81,7 @@ func TestProcessArgs(t *testing.T) {
 		[]string{"-sort-predicates"},
 		false,
 		nil,
-		[]*medium{{typ: inline, eskip: `PathRegexp("^/") && Host("^(www[.]example[.]org[.]?(:[0-9]+)?)$") -> "https://www.example.org"`}},
```

Comment:
This test essentially just tests that `-sort-predicates` flag exists. It fails if you run `go test ./cmd/eskip/` because expected media values do not match return value from `processArgs` being tested.